AttitudeManager/ControlAlgorithms/Src/AM_LevelQuadControl.cpp

```diff
@@ -0,0 +1,75 @@
+#include "AM_LevelQuadControl.hpp"
+#include "PID.hpp"
+
+// #include "LOS_SensorFusion.hpp" Whatever this is it will be needed
+typedef struct {
+    float roll, pitch, yaw;              // Degrees. Yaw of 180 is north.
+    float rollRate, pitchRate, yawRate;  // Degrees/second
+    float airspeed;                      // m/s
+    float altitude;                      // m
+    float rateOfClimb;                   // m/s
+    long double latitude;                // Decimal degrees
+    float latitudeSpeed;                 // m/s
+    long double longitude;               // Decimal degrees
+    float longitudeSpeed;                // m/s
+    double track;                        // Degrees. Track of 0 is north.
+    float groundSpeed;                   // m/s
+    double heading;                      // Degrees. Heading of 0 is north.
+} SFOutput_t;
```

Comment:
Will need to be removed once the los SF interface is ready to go, then we just grab this data from los